ckanext/xloader/templates/xloader/resource_data.html

```diff
@@ -5,10 +5,24 @@
 {% block primary_content_inner %}
 
   {% set action = h.url_for('xloader.resource_data', id=pkg.name, resource_id=res.id) %}
+  {% set delete_action = h.url_for('xloader.delete_datastore_table', id=pkg.id, resource_id=res.id) %}
   {% set show_table = true %}
 
+  {% block delete_ds_button %}
+    <form method="post" action="{{ delete_action }}" class="mb-3 d-inline-block">
+      {{ h.csrf_input() if 'csrf_input' in h }}
+      <a href="{{ h.url_for('xloader.delete_datastore_table', id=pkg.id, resource_id=res.id) }}"
+        class="btn btn-danger pull-left"
+        type="submit"
+        data-module="confirm-action"
+        data-module-with-data=true
+        data-module-content="{{ _('Are you sure you want to delete this DataStore table and Data Dictionary?') }}"
+        >{% block delete_datastore_button_text %}<i class="fa fa-remove"></i>{{ _('Delete DataStore table') }}{% endblock %}</a>
```

Comment:
Probably shouldn't include the word "table" here since it's not usually in the Datastore UI.

Reply:
Word "table" has been removed from user visible things here